drivers/CoreLED/tests/CoreLED_test_brightness.cpp

```diff
@@ -0,0 +1,257 @@
+// Leka - LekaOS
```

Comment:
ces tests sont très détaillés. Cela dit, je ne suis pas sûr qu'ils testent vraiment ce qu'on cherche.
on part du principe qu'on a la minimum_brightness au départ, mais ça n'a pas de sens, autant avoir la default.

du coup si on change l'implémentation, les tests deviennent tous bons sans avoir à utiliser setColor.

si on doit garder la minimum brightness au départ (et il faut dire pourquoi), dans ce cas là tu as 3 tests:

- qu'au début c'est bien minimum
- que si tu setColor, ça met bien au default
- que si tu fade to black, ça diminue bien

si au contraire on part de la valeur par défaut, dans ce qu'à là tu as deux tests:

- que le fade to black diminue bien
- qu'une fois le fade to black terminé, quand tu setColor, ça remonte.